bazel/re2c.bzl

```diff
@@ -0,0 +1,81 @@
+# re2c generator rules - assuming that re2c is available as '@re2c//:re2c'.
+
+visibility("public")
+
+def _re2c_impl(ctx):
+    srcs = ctx.files.srcs
+    if len(srcs) != 1:
+        fail("Attribute `srcs` must contain exactly 1 '.re2c' file.")
```

Comment:
The usual file extension is .re, not .re2c.

Reply:
Done.